l10n_de_tax_statement/models/l10n_de_tax_statement_2019.py

```diff
@@ -282,15 +282,65 @@ def _finalize_lines_2019(lines):
     # calculate line 51
     lines['51']['tax'] = lines['51']['tax'] * 1
     _51b = lines['51']['tax'] + _26b + _27b + _28b + _30b + _33b + _34b \
-           + _35b + _36b + _48b + _49b + _50b
+        + _35b + _36b + _48b + _49b + _50b
     # calculate line 60
     lines['60']['tax'] = lines['60']['tax'] * 1
     _60b = lines['60']['tax'] + _51b - _53b - _54b - _55b \
-           - _56b - _57b - _58b - _59b
+        - _56b - _57b - _58b - _59b
     # calculate line 64
     lines['64']['tax'] = lines['64']['tax'] * 1
     _64b = lines['64']['tax'] + _60b + _62b + _63b
     # update lines 51, 60, 64
     lines['64'].update({'tax': _64b})
     lines['51'].update({'tax': _51b})
     lines['60'].update({'tax': _60b})
+
+
+def _get_tags_map_2019(config):
+    return {
+        config.tag_41_base.id: ('20', 'base'),
```

Comment:
@tv-openbig could you double-check if the mapping tags-lines is correct?

Reply:
I have deleted some lines, as we have some deleted lines in the offical VAT formular (elsteronline.de portal). However i assume that we have to assign other tags to some taxes (removal of tax tags "68" -> now "60" // "52" and "78" -> now "84" // "53" and "79" -> now "85"). All taxes with those tags needs to be revised (however just from 01/01/2019 not before !!!). How should we address that ?

Reply:
@tv-openbig the lines you just removed https://github.com/OCA/l10n-germany/pull/39/commits/ba91812cb91b84434d8ca9320865a7f2ebf11ffa were instead doing the job:

- tag [68](https://github.com/OCA/l10n-germany/pull/39/commits/ba91812cb91b84434d8ca9320865a7f2ebf11ffa#diff-83ae6e5803d61484e5e11e55c1d1e4a1L323), mapping line 39 (same as tag 60)
- tags [52](https://github.com/OCA/l10n-germany/pull/39/commits/ba91812cb91b84434d8ca9320865a7f2ebf11ffa#diff-83ae6e5803d61484e5e11e55c1d1e4a1L329) and [78](https://github.com/OCA/l10n-germany/pull/39/commits/ba91812cb91b84434d8ca9320865a7f2ebf11ffa#diff-83ae6e5803d61484e5e11e55c1d1e4a1L333), mapping line 50 (same as tag 84)
- tags [53](https://github.com/OCA/l10n-germany/pull/39/commits/ba91812cb91b84434d8ca9320865a7f2ebf11ffa#diff-83ae6e5803d61484e5e11e55c1d1e4a1L330) and [79](https://github.com/OCA/l10n-germany/pull/39/commits/ba91812cb91b84434d8ca9320865a7f2ebf11ffa#diff-83ae6e5803d61484e5e11e55c1d1e4a1L334), mapping line 50 (same as tag 85)

Could you check?

Reply:
I have again added the deleted lines.